tasks.py

```diff
@@ -552,20 +552,33 @@ def lint(ctx, env="container"):
 
 
 @task(help={
-    "name": "name of the kind cluster to test.",
-    "export": "where to export kind logs"
+    "name": "name of the kind cluster to test (only kind uses).",
+    "export": "where to export kind logs.",
+    "kubeconfig_path": "kubeconfig path. By default, load kubeconfig from kind.",
+    "system_namespace": "comma separated kubernetes system namespace list.",
```

Comment:
Suggests 
"system_namespaces": "comma separated list of Kubernetes system namespaces"

Reply:
changed.